src/essence/Basics/Layers_/Layers_.js

```diff
@@ -1812,3 +2108,4 @@ function parseConfig(configData, urlOnLayers) {
 }
 
 export default L_
+window.L_ = L_
```

Comment:
Is this needed and where?

Reply:
Fixed 118971b12711ba13af5483224bc2295e9d8d0cbe!